components/CMakeLists.txt

```diff
@@ -236,6 +236,23 @@ if (UNIX AND NOT APPLE)
 target_link_libraries(components ${CMAKE_THREAD_LIBS_INIT})
 endif()
 
+add_library(osgdb_serializers_nifosg SHARED
+    serializers/nifloader serializers/controller
+    serializers/particle serializers/userdata
+    )
+
+target_link_libraries(osgdb_serializers_nifosg components)
+
+add_library(osgdb_serializers_sceneutil SHARED
```

Comment:
The serializer namespace has nothing to do with C++ namespaces, correct? In that case an OpenMW serializer namespace would be fine.